app/src/main/java/com/example/linkdevmvvm/utils/Constants.java

```diff
@@ -0,0 +1,11 @@
+package com.example.linkdevmvvm.utils;
+
+/**
+ * Created by antonio on 1/16/19.
+ */
+
+public class Constants {
+    public final static String BASE_URL = "https://newsapi.org/v1/";
+    public final static String SOURCE = "the-next-web";
+    public final static String API_KEY = "533af958594143758318137469b41ba9";
```

Comment:
The Api key should be added in the gradle file same as [here](https://medium.com/code-better/hiding-api-keys-from-your-android-repository-b23f5598b906).

Reply:
Link not open but I added it in gradle file